AccountingSystem/Client/Types/BaseAccountingType.py

```diff
@@ -42,7 +38,7 @@ def checkType( self ):
     self.fieldsList.extend( self.keyFieldsList )
     self.fieldsList.extend( self.valueFieldsList )
     if len( self.valuesList ) != len( self.fieldsList ):
-      self.valuesList = [ None for i in self.fieldsList ]
+      self.valuesList = [ None for _i in self.fieldsList ]
```

Comment:
Really minor, but I suppose that ``` [None] * len(self.fieldsList) ``` is faster and more memory efficient. But we do not really care for 4 elements I suppose :)